static/webui/script.js

```diff
@@ -701,3 +701,83 @@ function doChangeSettings() {
             });
         });
 }
+
+// Cheats route
+
+fetch("http://localhost:61558/ping", { mode: "no-cors" }).then(() => {
+    $("#client-cheats-nok").addClass("d-none");
+    $("#client-cheats-ok").removeClass("d-none");
+
+    fetch("http://localhost:61558/skip_mission_start_timer")
+        .then(res => res.text())
+        .then(res => {
+            document.getElementById("skip_mission_start_timer").checked = res == "1";
+        });
+    document.getElementById("skip_mission_start_timer").onchange = function () {
+        fetch("http://localhost:61558/skip_mission_start_timer?" + this.checked);
+    };
+
+    fetch("http://localhost:61558/fov_override")
+        .then(res => res.text())
+        .then(res => {
+            document.getElementById("fov_override").value = parseFloat(res) * 10000;
+        });
+    document.getElementById("fov_override").oninput = function () {
+        fetch("http://localhost:61558/fov_override?" + this.value);
+    };
+});
+
+function doUnlockAllFocusSchools() {
```

Comment:
in general I noticed you like using the do prefix, however I think it doesn't add to the function name. 
Just something I personally would change.

Reply:
I use this prefix for functions that are invoked from the UI as opposed to internal functions. I think it somewhat clarifies expectations.